lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.t

```diff
@@ -51,6 +54,7 @@ my $target_project = Genome::Config::AnalysisProject->create(
 my $cmd = $class->create(
     to_project => $target_project,
     from_project => $source_project,
+    tags => 1,
```

Comment:
Seems like maybe it would be overkill but you might test that tags aren't copied in the opposite case and/or that the command works okay if no tags exist.